meson.build

```diff
@@ -232,11 +232,15 @@ if get_option('enable-python')
   endif
 endif
 
-# nnfw ( details in https://review.tizen.org/gerrit/p/platform/core/ml/nnfw )
+# nnfw-runtime ( details in https://review.tizen.org/gerrit/p/platform/core/ml/nnfw )
+if get_option('enable-nnfw-runtime')
+  add_project_arguments('-DENABLE_NNFW_RUNTIME=1', language: ['c', 'cpp'])
+  # This also enables nnfw as a subplugin for Tizen.
+endif
+
 if get_option('enable-nnfw')
   add_project_arguments('-DENABLE_NNFW=1', language: ['c', 'cpp'])
-  # For tf-lite/nnapi, enable-nnfw allows to use nnfw as a backend.
-  # This also enables nnfw as a subplugin for Tizen.
+  # For tf-lite/nnapi, enable-nnfw allows to use nnfw::tflite::nnapi as a backend.
```

Comment:
Uh... both options are referring to https://review.tizen.org/gerrit/p/platform/core/ml/nnfw
Check its packages (sub-packages) and see if which is for NNAPI and which is for an independent runtime. It is possible that a single package has both roles.

Reply:
Yes, you are right. Both options are referring to https://review.tizen.org/gerrit/p/platform/core/ml/nnfw
However, as we discussed, one option can be set to enable nnfw as backend in tf-lite filter (now `ENABLE_NNFW_TFLITE`), and the other option can be set to enable nnfw as a tensor filter element (`ENABLE_NNFW_RUNTIME`).

Reply:
Please suggest better naming if possible.

Reply:
If we do not or cannot specify backend of NNAPI, it should be enable-tensorflow-lite-nnapi (if tf-lite is the only user) or enable-nnapi (if others are going to use it).

If we can and do specify backend of NNAPI, we need something like nnapi-backend=(string) [ nnfw-runtime, armcl, none / disabled, ... ]

Reply:
I cannot find an option to specify backend of NNAPI as of now. 
Currently, tf-lite is the only user in nnstreamer, but caffe2 also supports NNAPI. So, `enable-nnapi` is better, but this option will just enable nnapi and already exists in [nnstreamer.ini.in](https://github.com/nnsuite/nnstreamer/blob/ea2ddd13bcaabc9ed38cf68bb846cb5f69a0bb49/nnstreamer.ini.in#L16).

However, `ENABLE_NNFW_TFLITE` is supposed to enable nnapi with nnfw backend.

Reply:
If we cannot specify backend, then, let's settle with ENABLE_TFLITE_NNAPI. In such cases, nnfw is meaningless.

Anyway, please note that we need to confirm that nnfw(NNAPI-backend) is working with tflite when nnapi is enabled in Tizen. (but other any backends available in other platforms).

Reply:
> If we cannot specify backend, then, let's settle with ENABLE_TFLITE_NNAPI. In such cases, specifying nnfw is meaningless.

Yes we cannot specify backend in nnapi. However, when ENABLE_NNFW_TFLITE is set (or as you suggested ENABLE_TFLITE_NNAPI), interpreter provided by nnfw is used instead of nnapi's interpreter. So, because backend cannot be specified in nnapi, different interpreter is used. 
Given this, I think ENABLE_TFLITE_NNAPI naming can be confusing as this macro enables uses nnfw interpreter instead of nnapi interpreter (counter to the name which suggests enabling of nnapi in tflite).

> Anyway, please note that we need to confirm that nnfw(NNAPI-backend) is working with tflite when nnapi is enabled in Tizen. (but other any backends available in other platforms).

yes.

Reply:
Whether it's using nnapi interpreter or nnfw interpreter, with this macro enabled, it uses tensorflow-lite as .tflite parser and nnfw as tf-lite engine. For users of nnstreamer, does the entity that provides nnapi interpreter matter?

In the code affected by this macro, is there any non-tf-lite, but nnfw related code? It only appears to enable "delegation", which I assume it is trying to delegate "nnapi interpretation".

Assuming that another backend is available and we can use this option (in other words, we can let another backend to interprete nnapi instead of both tf-lite and nnfw), having nnfw in the option name may mislead. If activating nnapi is not affected by this macro, but only whether how interpreter is enabled is affected, the name (both option and macro) may need to reflect it: e.g., ENABLE_TFLITE_NNAPI_DELEGATION or ENABLE_TFLITE_NNAPI_INTERPRETER_DELEGATION or something else)

Reply:
Yes, updated to ENABLE_TFLITE_NNAPI_DELEGATION